topology/mask.h

```diff
@@ -210,12 +210,17 @@ class BoxMask : public Mask< D >
 public:
   /**
    * Parameters that should be in the dictionary:
-   * lower_left  - Position of lower left corner (array of doubles)
-   * upper_right - Position of upper right corner (array of doubles)
+   * lower_left    - Position of lower left corner (array of doubles)
+   * upper_right   - Position of upper right corner (array of doubles)
+   * azimuth_angle - Rotation angle in degrees from x-axis (double), optional
```

Comment:
Is it documented somewhere that the angles are in degrees? Maybe also add that the polar angle does not apply to 2D.

Reply:
I have now added a note about the angles in the python documentation for masks.